lib/ViewModels/ChartPanelViewModel.js

```diff
@@ -49,6 +57,16 @@ var ChartPanelViewModel = function(options) {
 
     updateMaxDimensions();
 
+    knockout.getObservable(this, 'isVisible').subscribe(function(isVisible) {
+        var cesiumContainer = this.terria.currentViewer.viewer.container;
```

Comment:
I can do it... actually I'm thinking that we should really define a `getContainer` method on `currentViewer` directly, and then use that.